webapp/actions/global_actions.jsx

```diff
@@ -140,15 +141,15 @@ export function doFocusPost(channelId, postId, data) {
         channelId,
         post_list: data
     });
-    AsyncClient.getChannels(true);
+    loadChannelsForCurrentUser();
```

Comment:
Probably not, but I didn't want to go removing a bunch of stuff this close to code complete. Ticket [here](https://mattermost.atlassian.net/browse/PLT-4594) to look at this later.